wiki/Tournaments/OMF/WC/en.md

```diff
@@ -0,0 +1,485 @@
+| Finals | 2021-09-25/2021-09-26 |
+| Grand Finals | 2021-10-02/2021-10-03 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, profile banner |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | Profile banner |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | Profile banner |
+
+
+## Organisation
+
+The osu! World Cup 2020 was run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519) |
+| Mappool selector | ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_US] [141](https://osu.ppy.sh/users/10193533), ![][flag_PL] [-Sylvari](https://osu.ppy.sh/users/3493804), ![][flag_KR] [isome](https://osu.ppy.sh/users/87546), ![][flag_US] [Flux3on](https://osu.ppy.sh/users/9401932), ![][flag_US] [Aery](https://osu.ppy.sh/users/3994671),![][flag_FI] [Nyanaro](https://osu.ppy.sh/users/4157611)  |
+| Mappool playtester | ![][flag_ID] [rHO](https://osu.ppy.sh/users/1629553), ![][flag_NO] [-GN](https://osu.ppy.sh/users/895581), ![][flag_FR] [Rulue](https://osu.ppy.sh/users/3723612), ![][flag_CA] [ktgster](https://osu.ppy.sh/users/53378), ![][flag_US] [semaphore](https://osu.ppy.sh/users/6313643) |
+| Commentator | ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519), ![][flag_AU] [xtremeities](https://osu.ppy.sh/users/10759664), ![][flag_US] [Daprin](https://osu.ppy.sh/users/10961983), ![][flag_KR] [isome](https://osu.ppy.sh/users/87546), ![][flag_RU] [Prade](https://osu.ppy.sh/users/9318565), ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983), ![][flag_NL] [n0ah](https://osu.ppy.sh/users/3086393) |
+| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_SG] [uchuuj1n](https://osu.ppy.sh/users/9140302), ![][flag_US] [Rekunan](https://osu.ppy.sh/users/8203119), ![][flag_VN] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_PL] [P a t r i c k](https://osu.ppy.sh/users/6814521), ![][flag_DE] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_SI] [Redavor](https://osu.ppy.sh/users/3328606), ![][flag_PL] [Kondi](https://osu.ppy.sh/users/7382321), ![][flag_US] [freddiiieeee](https://osu.ppy.sh/users/7112839), ![][flag_KZ] [brwn31](https://osu.ppy.sh/users/10379418) |
+| Statistician |  ![][flag_FR] [Kaeldori](https://osu.ppy.sh/users/962519) |
+| GFX |  ![][flag_DE] [Suilerua](https://osu.ppy.sh/users/13137699) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1359817)
+- [Livestream](https://www.twitch.tv/Kaeldori "Twitch")
+- [Challonge bracket](https://challonge.com/OMFWC "Challonge")
+- [Pick'ems page](https://pickem.hwc.hr/tournaments/72 "hwchr's pick'em") hosted by ![][flag_DE] [hallowatcher](https://osu.ppy.sh/users/1874761)
+- **[Main sheet](https://docs.google.com/spreadsheets/d/1YwFNdAnKQL4jCqZQmxUGc-Q_5hHj1fdhOdbTcWKZhsw/edit#gid=0 "Google Sheets")**
+
+## Participants
```

Comment:
My assumption here is that the participants list is sorted by seed rather than by alphabetical order. My preference would be to change the table to be in alphabetical order to match with other would cup style tournaments.

Reply:
Fixed